apps/src/p5lab/P5Lab.js

```diff
@@ -319,7 +319,8 @@ P5Lab.prototype.init = function(config) {
 
     // Store p5specialFunctions in the unusedConfig array so we don't give warnings
     // about these functions not being called:
-    config.unusedConfig = this.p5Wrapper.p5specialFunctions;
+    // Clone p5specialFunctions so we can remove 'setup' from unusedConfig but not p5specialFunctions
+    config.unusedConfig = this.p5Wrapper.p5specialFunctions.slice(0);
```

Comment:
Also, could do `config.unusedConfig = [...this.p5Wrapper.p5specialFunctions];`

Reply:
Ah I forgot about that syntax, that seems cleaner thanks!